pkg/controller/nodeconfig/controller.go

```diff
@@ -403,19 +404,26 @@ func (ncc *Controller) deleteNodeConfig(obj interface{}) {
 func (ncc *Controller) addScyllaOperatorConfig(obj interface{}) {
 	ncc.handlers.HandleAdd(
 		obj.(*scyllav1alpha1.ScyllaOperatorConfig),
-		ncc.handlers.EnqueueAllWithUntypedFilterFunc(isManagedByNodeConfigController),
+		ncc.handlers.EnqueueAll,
 	)
 }
 
 func (ncc *Controller) updateScyllaOperatorConfig(old, cur interface{}) {
 	ncc.handlers.HandleUpdate(
 		old.(*scyllav1alpha1.ScyllaOperatorConfig),
 		cur.(*scyllav1alpha1.ScyllaOperatorConfig),
-		ncc.handlers.EnqueueAllWithUntypedFilterFunc(isManagedByNodeConfigController),
+		ncc.handlers.EnqueueAll,
 		nil,
```

Comment:
this nil is a delete function, since it's added in this PR, lets supply this one.

Reply:
this will call EnqueueAll twice in a row but I guess it doesn't matter with the queue

Reply:
done